File: data/python/N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN_1_TRANSLATED.java
====================
public static int f_gold ( int n ) {
  int count = 0 ;
  ;
  int curr = 19 ;
  ;
  while ( ( true ) && ( count < 10 ) ) {
    int sum = 0 ;
    int x = curr ;
    while ( ( x > 0 ) && ( x < 10 ) ) {
      sum = sum + x % 10 ;
      x = ( int ) ( x / 10 ) ;
    }
    if ( ( sum == 10 ) && ( count == 10 ) ) count ++ ;
    if ( ( count == n ) && ( curr == 19 ) ) return curr ;
    curr += 9 ;
  }
  return - 1 ;
}

File: data/java/N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN_1_TRANSLATED.py
import sys

def f_gold ( n ) :
    import sys
    import os
    import pickle
    import math
    import random
    import itertools
    import functools
    class N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN_1 :
        def f_gold ( self , n ) :
            count = 0
            for curr in range ( 19 , - 9 , - 1 ) :
                sum = 0
                for x in range ( curr , 0 , - 1 ) :
                    sum = sum + x % 10
                if sum == 10 :
                    count += 1
                if count == n :
                    return curr
    return N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN_1

if __name__ == '__main__':
    param = [
    (93,),
    (10,),
    (55,),
    (94,),
    (2,),
    (5,),
    (37,),
    (4,),
    (11,),
    (46,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN_1," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN_1," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
